charts/opensearch-operator/values.yaml

```diff
@@ -3,11 +3,32 @@ tolerations: []
 manager:
   resources:
     limits:
-      cpu: 2
-      memory: 1Gi
+      cpu: 200m
+      memory: 300Mi
     requests:
-      cpu: 2
-      memory: 1Gi
+      cpu: 100m
```

Comment:
did you check that numbers? cause as i remember the operator is not working well with under than 1 cpu and 1gb mem

Reply:
@idanl21 In our deployment, the Operator idles at usages of ~275 MB RAM and 10 Mi CPUs with spikes of ~370 MB RAM and 20 Mi CPUs. I raised the memory requests & limits a little bit to match that 👍